console/console-init/ui/src/modules/project-detail/utils/constant.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import { ISelectOption } from "utils";
+
+export const algorithmTypeOptions: ISelectOption[] = [
+  { key: "RSA", label: "RSA", value: "RSA" },
```

Comment:
try to keep keys in small case i.e. key:"rsa", key:"ec"

Reply:
done